include/aikido/util/detail/yaml_extension.hpp

```diff
@@ -135,13 +131,13 @@ struct convert<Eigen::
       throw YAML::RepresentationException(getMark(node), ss.str());
     }
 
-    if (node.Tag() == "!Vector")
+    if (node[0].Type() == YAML::NodeType::Scalar)
     {
       matrix.resize(rows, 1);
       for (Index i = 0; i < rows; ++i)
         matrix(i, 0) = node[i].template as<_Scalar>();
```

Comment:
Do we want to check that _each_ row of the sequence is also a `Scalar`.  Otherwise, if we try to load a sequence like `[2, [2,3], 4, 5]` we will potentially get an unhelpful error message like `YAML::BadConversion`.  I am not sure if this is the case or not.